test/dummy/app/components/header/header.js

```diff
@@ -1 +1,3 @@
-console && console.log('header')
+if(typeof console !== "undefined") {
+  console.log('header')
```

Comment:
Mixed double and single quotes.<br>Missing semicolon.<br>'console' is not defined.